README.md

```diff
@@ -58,7 +58,7 @@ Functions and Features:
 | pt-br | Português (Brasil) | [Hélio de Souza](https://tinyurl.com/hdstec), [Igor Rückert](https://github.com/igorruckert) |
 | ro-ro | Română (România) | ianymaty |
 | ru-ru | Русский (Россия) | [Долматов Алексей](https://github.com/iDolmatov) |
-| th-th | ไทย (ไทย) | OllyMaster, Charon |
+| th-th | ไทย (ไทย) | [OllyMaster] (https://github.com/OllyMaster), Charon |
```

Comment:
```suggestion
| th-th | ไทย (ไทย) | [OllyMaster](https://github.com/OllyMaster), Charon |
```

Hi, @OllyMaster . It seems that you might want to add a hyperlink. The correct format is `[text](link)` , there is no space between the right bracket and the left parenthesis.

Reply:
You're welcome. I just gave a suggestion, **YerongAI** fixed it in a6d40aef34d04cd0dbcff77dca092cdd6c5d1e62 .